docs/XBlocks.md

```diff
@@ -0,0 +1,217 @@
+# Group Project building blocks
+
+The following blocks are a part of Group Project XBlock v2.
+
+## Top-level XBlocks
+
+* ` Group Project` - top level XBlock, represents entire group project as a whole
+    * Settings:
+        * `Display Name` - human-friendly name for Group Project - displayed in admin console.
```

Comment:
@e-kolpakov What's the difference between "admin console" and "admin dashboard"?